src/amo/components/Categories.scss

```diff
@@ -17,4 +16,17 @@
   display: block;
   padding: 1em 0;
   text-decoration: none;
+
+  &::before {
+    background: url('../img/icons/arrow.svg') 100% 50% no-repeat;
+    content: '';
+    float: right;
```

Comment:
I think if you set `display` to `inline-block` it might negate the need for floats.